action.yml

```
@@ -9,6 +9,10 @@ inputs:
     description: 'The GITHUB_TOKEN secret'
     required: false
     default: ${{ github.token }}
+  github-api-url:
+    description: 'Github API URL'
+    required: true
+    default: 'https://api.github.com'
```

```suggestion
    default: '${{ github.api_url }}'
```

Can we use placeholder for getting default value?